### PR TITLE
add data feed swagger docs

### PR DESCRIPTION
**Pull request by satellite15**
Signed-off-by: Martin Scott <mscott@chef.io>

### :nut_and_bolt: Description: What code changed, and why?

Swagger docs added

### :white_check_mark: Checklist

- [x] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [ ] Tests added/updated? NA
- [x] Docs added/updated?
- [x] All commits have been signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

### CODE REVIEW

File: api/external/data_feed/data_feed.proto [hunk 1]
```diff
@@ -36,6 +76,19 @@ service DatafeedService {
 			action: "destination:destination:delete"
 		};
 	};
+/*
+  Update a datafeed destination
+
+  Updates a datafeed destination.
```
Comment (mjingle) [inline api/external/data_feed/data_feed.proto:82]: @satellite15 Does this action update the datafeed destination ID or other details that does not include the datafeed destination ID? 

Related: Can you update more than one detail at a time with this action?

Comment (mjingle) [reply api/external/data_feed/data_feed.proto]: ^ I still have these two questions, but I edited this content with my current interpretation, which can always be changed.

Comment (satellite15) [reply api/external/data_feed/data_feed.proto]: You can modify more than one detail, I just tested modifying the id and it doesn't work.

File: api/external/data_feed/data_feed.proto [hunk 2]
```diff
@@ -36,6 +77,17 @@ service DatafeedService {
 			action: "destination:destination:delete"
 		};
 	};
+/*
+  Update a datafeed destination
+
+  Updates a datafeed destination's `name`, `url` or `secret`.
```
Comment (mjingle) [inline api/external/data_feed/data_feed.proto:83]: @satellite15 would this updated interpretation be correct then?

Comment (satellite15) [reply api/external/data_feed/data_feed.proto:83]: yes

**Comment by mjingle** [comment]
@satellite15 Thank you so much for this API documentation so far! Could you please add tags and tag groups to these entries? Our internal documentation on tags and tag groups is available here: https://github.com/chef/automate/tree/master/components/automate-chef-io#tags-and-tag-groups

**Comment by mjingle** [comment]
This content needs to be reopened in the new docs directory when needed.

**Pull request closed without merging**